### PR TITLE
Prepare idr0044 for import

### PR DESCRIPTION
**Pull request by dominikl**
This PR adds the files needed for the import and annotation of the idr0044 project (annotation.csv, bulkmap config, etc.). 

I also included all the original files which have been submitted (see 'submission' folder). Took me a while to get all files together from the different sources. What are your thoughts on that @sbesson @francesw ? Can remove it again if needed. 

Not quite ready yet: Still need to assemble the study.txt from the four study descriptions provided.

### CODE REVIEW

File: experimentA/idr0044-experimentA-filePaths.tsv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Dataset1	/uod/idr/filesets/idr0044-mcdole/20180412-ftp/Dataset1/ImageData/
-140813	/uod/idr/filesets/idr0044-mcdole/20180926-ftp/140813/
-151205	/uod/idr/filesets/idr0044-mcdole/20180926-ftp/151205/
-160723	/uod/idr/filesets/idr0044-mcdole/20180926-ftp/160723/
+Embryo1	/uod/idr/filesets/idr0044-mcdole/20180412-ftp/Dataset1/ImageData/
```
Comment (sbesson) [inline experimentA/idr0044-experimentA-filePaths.tsv]: Following the renaming `idr0044-mcdole` will need to be renamed as `idr0044-mcdole-tardislightsheet`

File: submission/idr0000_KellerLab.txt [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+Study Organism	"""Mus musculus"""			
+Study Organism Term Source REF	NCBITaxon 10090 			
+Study Organism Term Accession	# leave blank			
+Study Experiments Number	1			
+Study External URL	# if there is an existing web page related to the study enter it here			
+Study Public Release Date	"# enter a date of when the data can be made public, if known"			
+				
+# Study Publication				
+Study PubMed ID	# fill in if known
```
Comment (sbesson) [inline submission/idr0000_KellerLab.txt]: 30318151

File: submission/idr0000_KellerLab.txt [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+Study Organism	"""Mus musculus"""			
+Study Organism Term Source REF	NCBITaxon 10090 			
+Study Organism Term Accession	# leave blank			
+Study Experiments Number	1			
+Study External URL	# if there is an existing web page related to the study enter it here			
+Study Public Release Date	"# enter a date of when the data can be made public, if known"			
+				
+# Study Publication				
+Study PubMed ID	# fill in if known			
+Study Publication Title	# fill in if known
```
Comment (sbesson) [inline submission/idr0000_KellerLab.txt]: Assuming the same as the `Study Title`?

File: submission/idr0000_KellerLab.txt [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+Study Experiments Number	1			
+Study External URL	# if there is an existing web page related to the study enter it here			
+Study Public Release Date	"# enter a date of when the data can be made public, if known"			
+				
+# Study Publication				
+Study PubMed ID	# fill in if known			
+Study Publication Title	# fill in if known			
+Study Author List	"Katie McDole, Léo Guignard, Fernando Amat, Andrew Berger, Grégoire Malandain, Loïc A. Royer, Srinivas C. Turaga, Kristin Branson, Philipp J. Keller"			
+Study PMC ID	# fill in if known			
+Study DOI	# fill in if known
```
Comment (sbesson) [inline submission/idr0000_KellerLab.txt]: https://doi.org/10.1016/j.cell.2018.09.031

File: submission/idr0000_KellerLab.txt [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+				
+# Study Contacts				
+Study Person Last Name	Keller	Kate	Leo	
+Study Person First Name	Philipp	McDole	Guignard	
+Study Person Email	kellerp@janelia.hhmi.org	mcdolek@janelia.hhmi.org	guignardl@janelia.hhmi.org	
+Study Person Address	19700 Helix Dr. Ashburn Va 20147 USA	19700 Helix Dr. Ashburn Va 20147 USA	19700 Helix Dr. Ashburn Va 20147 USA	
+Study Person Roles	submitter	submitter	submitter	
+				
+# Study License and Data DOI				
+Study License	# leave blank
```
Comment (sbesson) [inline submission/idr0000_KellerLab.txt]: This is something we might need to clarify with Kate et al i.e. are they fine with making the data `CC-BY 4.0` with copyright `McDole et al.` ? /cc @francesw

**Comment by dominikl** [comment]
I have no idea how I can link the annotations to the 16 imported images, which are:
```
Dataset "Embryo1"; 1 fileset with 4 images:
SPM00_TM000512_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb [SPM00_TM000512_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb #1]
SPM00_TM000512_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb [SPM00_TM000512_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb #2]
SPM00_TM000512_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb [SPM00_TM000512_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb #3]
SPM00_TM000512_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb [SPM00_TM000512_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb #4]
   
Dataset "Embryo2"; 1 fileset with 4 images:             
SPM00_TM000184_CM00_CM01_CHN00.fusedStack_xyProjection.corrected.shifted.klb [SPM00_TM000184_CM00_CM01_CHN00.fusedStack_xyProjection.corrected.shifted.klb #1]
SPM00_TM000184_CM00_CM01_CHN00.fusedStack_xyProjection.corrected.shifted.klb [SPM00_TM000184_CM00_CM01_CHN00.fusedStack_xyProjection.corrected.shifted.klb #2]
SPM00_TM000184_CM00_CM01_CHN00.fusedStack_xyProjection.corrected.shifted.klb [SPM00_TM000184_CM00_CM01_CHN00.fusedStack_xyProjection.corrected.shifted.klb #3]
SPM00_TM000184_CM00_CM01_CHN00.fusedStack_xyProjection.corrected.shifted.klb [SPM00_TM000184_CM00_CM01_CHN00.fusedStack_xyProjection.corrected.shifted.klb #4]
        
Dataset "Embryo3"; 1 fileset with 4 images:        
SPM00_TM000319_CM00_CM01_CHN00.fusedStack_yzProjection.corrected.shifted.klb [SPM00_TM000319_CM00_CM01_CHN00.fusedStack_yzProjection.corrected.shifted.klb #1]
SPM00_TM000319_CM00_CM01_CHN00.fusedStack_yzProjection.corrected.shifted.klb [SPM00_TM000319_CM00_CM01_CHN00.fusedStack_yzProjection.corrected.shifted.klb #2]
SPM00_TM000319_CM00_CM01_CHN00.fusedStack_yzProjection.corrected.shifted.klb [SPM00_TM000319_CM00_CM01_CHN00.fusedStack_yzProjection.corrected.shifted.klb #3]
SPM00_TM000319_CM00_CM01_CHN00.fusedStack_yzProjection.corrected.shifted.klb [SPM00_TM000319_CM00_CM01_CHN00.fusedStack_yzProjection.corrected.shifted.klb #4]
            
Dataset "Embryo4"; 1 fileset with 4 images:    
SPM00_TM000136_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb [SPM00_TM000136_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb #1]
SPM00_TM000136_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb [SPM00_TM000136_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb #2]
SPM00_TM000136_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb [SPM00_TM000136_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb #3]
SPM00_TM000136_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb [SPM00_TM000136_CM00_CM01_CHN01.fusedStack.corrected.shifted.klb #4]
```

The provided annotation spreadsheet has "Image Names" like
``` SPM00_TM000245_CM00_CM01_CHN00_xyProjection.klb ```

So there's no relation from the image name mentioned in the annotations with the image name of the imported images. And the imported images are part of filesets, each image of a fileset references the same "imported" files, so this neither can be used to relate the annotations to the imported images.

**Comment by francesw** [comment]
[All_Experiment_Keller2.xlsx](https://github.com/IDR/idr0044-mcdole-tardislightsheet/files/2553782/All_Experiment_Keller2.xlsx)
Added old image file names to match new image file names for annotations.

**Comment by francesw** [comment]
[All_Experiment_Keller2.xlsx](https://github.com/IDR/idr0044-mcdole-tardislightsheet/files/2562485/All_Experiment_Keller2.xlsx)
Amended shortened image names to reflect old image names.

**Comment by sbesson** [comment]
Looks good. Merging so that we can load this into `test60` and we can iterate from there.

**Pull request merged**